core/src/components/refresher/refresher.tsx

```diff
@@ -729,7 +729,15 @@ export class Refresher implements ComponentInterface {
         scrollStyle.transform = backgroundStyle.transform = y > 0 ? `translateY(${y}px) translateZ(0px)` : '';
         scrollStyle.transitionDuration = backgroundStyle.transitionDuration = duration;
         scrollStyle.transitionDelay = backgroundStyle.transitionDelay = delay;
-        scrollStyle.overflow = overflowVisible ? 'hidden' : '';
+        /**
+         * The goal is to avoid overriding the settings of certain virtual scroll
+         * libraries that already set the overflow style on the scroll container.
+         */
+        if (overflowVisible) {
+          scrollStyle.overflow = 'hidden';
+        } else if (scrollStyle.overflow === 'hidden') {
```

Comment:
I don't understand how this fixes the issue. I tried the reproduction you provided in https://github.com/ionic-team/ionic-framework/issues/26553#issuecomment-1385754385 with iOS mode, and the scroller is still disabled after completing the pull to refresh action.

My understanding of the issue is that we are overriding an `overflow` style that is set inline on the scroller (as opposed to in a stylesheet). If that's the case, we likely need to keep track of the inline style before we call `setCss` so we can restore the correct style.